### DISCUSSION TITLE
Hosting comments on GitHub for static websites

### DISCUSSION THREAD

**Discussion by daniele-salvagni** [daniele-salvagni/dan.salvagni.io#3] [Comments]

**Comment by daniele-salvagni** [comment]
Random comment to test this! 🐙

**Comment by zipang** [comment]
That's a pretty nice hack ! :smirk_cat:
Is the whole site automatically regenerated when a new issue is posted ?

**Comment by zipang** [comment]
Well.. it is ! 👏 :stuck_out_tongue_winking_eye: It is dynamically added.

**Comment by daniele-salvagni** [comment]
Yes, comments are loaded in Javascript each time the page loads, so they will always be up to date.

**Comment by douglasb78** [comment]
It's a very clever way to add comments to a website without having hosting costs or third-party services...

**Comment by daniele-salvagni** [comment]
I remade this website from scratch and I'm now using [Giscus](https://giscus.app/) which wasn't available at the time. I just had to convert the old issues to the newer Github discussions to migrate the old comments.
